CHEWBBACA/CHEWBBACA_NS/load_schema.py

```diff
@@ -0,0 +1,1352 @@
+      the necessary data has been uploaded. The Chewie-NS automatically
+      detects that all data has been received and finishes data insertion.
+
+Expected input
+--------------
+
+The process expects the following variables wheter through command line
+execution or invocation of the :py:func:`main` function:
+
+- ``-i``, ``input_files`` : Path to the directory of the schema to upload.
+
+    - e.g.: ``/home/user/schemas/ypestis_schema``
+
+- ``-sp``, ``species_id`` : The integer identifier or name of the species that
+  the schema will be associated to in the Chewie-NS.
+
+    - e.g.: ``1`` or ``'Yersinia pestis'``
+
+- ``-sn``, ``schema_name`` : A brief and meaningful name that should help
+  understand the type and content of the schema.
+
+    - e.g.: ``ypestis_cgMLST`` or ``ypestis cgMLST``
+
+- ``-lp``, ``loci_prefix`` : Prefix included in the name of each locus of the
+  schema.
+
+    - e.g.: ``ypestis``
+
+- ``--df``, ``description_file`` : Path to a text file with a description
+  about the schema. Markdown syntax is supported in order to allow greater
+  customizability of the rendered description in the Frontend (default='').
+
+    - e.g.: ``/home/user/schemas/ypestis_description``
+
+- ``--a``, ``annotations`` : Path to a TSV file with loci annotations. The
+  first column has loci identifiers (w/o .fasta extension), the second has
+  user annotations and the third has custom annotations (default=None).
+
+    - e.g.: ``/home/user/schemas/ypestis_annotations``
+
+- ``--cpu``, ``cpu_cores`` : Number of CPU cores that will be used in the
+  Schema Pre-processing step (default=1).
+
+    - e.g.: ``4``
+
+- ``--thr``, ``threads`` : Number of threads to use to search for annotations
+  on UniProt (default=20).
+
+    - e.g.: ``20``
+
+- ``--ns_url``, ``nomenclature_server_url`` : The base URL for the
+  Nomenclature Server.
+
+    - e.g.: ``http://127.0.0.1:5000/NS/api/`` (localhost)
+
+- ``--continue_up`` : If the process should check if the schema upload was
+  interrupted and try to resume it. ``True`` if provided, ``False`` otherwise.
+
+Code documentation
+------------------
+"""
+
+
+import os
+import sys
+import csv
+import json
+import pickle
+import argparse
+import requests
+import itertools
+import datetime as dt
+import multiprocessing
+import concurrent.futures
+
+from Bio import SeqIO
+from SPARQLWrapper import SPARQLWrapper, JSON
+from urllib3.exceptions import InsecureRequestWarning
+
+try:
+    from utils import constants as cnst
+    from utils import auxiliary_functions as aux
+    from utils import parameters_validation as pv
+except:
+    from CHEWBBACA.utils import constants as cnst
+    from CHEWBBACA.utils import auxiliary_functions as aux
+    from CHEWBBACA.utils import parameters_validation as pv
+
+
+# Suppress only the single warning from urllib3 needed.
+requests.packages.urllib3.disable_warnings(category=InsecureRequestWarning)
```

Comment:
The `InsecureRequestWarning` suppression can commented out because the warning will only be raised in local Chewie-NS instances with self-signed certificates. The same can be applied to the the warning import.